### PR TITLE
CurseForge Modpack

### PR DESCRIPTION
**Pull request by Doges**
bruh, github just closed my pr

### CODE REVIEW

File: src/commands/games/minecraft/modpack.js [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+const moment = require('moment')
+
+module.exports = class MinecraftModpack extends SearchCommand {
+  constructor (client) {
+    super({
+      name: 'modpack',
+      aliases: ['curseforge', 'm', 'twitch'],
+      parent: 'minecraft',
+      embedColor: Constants.CURSEFORGE_COLOR,
+      embedLogoURL: 'https://minecraft-hosting.com/wp-content/uploads/2019/01/curseforge.png'
```
Comment (jaxxibae) [inline src/commands/games/minecraft/modpack.js:11]: Please refrain from using third party websites. At best, [upload the image to Imgur](https://imgur.com)

File: src/commands/games/minecraft/modpack.js [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+  searchResultFormatter (items) {
+    return `[${items.name}](${items.websiteUrl})`
+  }
+
+  async handleResult ({ t, channel, author, language }, { name, websiteUrl, summary, attachments, downloadCount, dateReleased, latestFiles }) {
+    moment.locale(language)
+    channel.send(
+      new SwitchbladeEmbed(author)
+        .setThumbnail(attachments[0].url)
+        .setAuthor(name, 'https://minecraft-hosting.com/wp-content/uploads/2019/01/curseforge.png', websiteUrl)
```
Comment (jaxxibae) [inline src/commands/games/minecraft/modpack.js:28]: Same thing here. I think you can use `this.embedLogoURL`

**Review by jaxxibae — DISMISSED** [review]

File: src/commands/games/minecraft/modpack.js (hunk 1 above)
Comment (Doges) [reply src/commands/games/minecraft/modpack.js:11]: I mean, imgur is a third party website

File: src/locales/en-US/commands.json [hunk 3]
```diff
@@ -994,6 +994,13 @@
         "unknownName": "Name or UUID not found.",
         "title": "{{name}}'s name history!",
         "originalname": "Original Name"
+      },
+      "modpack": {
+        "commandDescription": "Shows a Minecraft modpack from CurseForge (Twitch).",
+        "commandUsage": "<modpack name>",
+        "downloadCount": "Download Count",
+        "releasedAt": "Released At",
+        "directDownload": "Direct Download"
```
Comment (metehus) [inline src/locales/en-US/commands.json:1003]:
```suggestion
        "downloadCount": "Downloads",
        "releasedAt": "Released at",
        "directDownload": "Direct download"
```

**Review by metehus — DISMISSED** [review]

**Review by almeidx — DISMISSED** [review]

**Review by almeidx — DISMISSED** [review]

**Review by davipatury — APPROVED** [review]
LGTM

File: src/apis/CurseForge.js [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+    })
+  }
+
+  searchAddon (gameId, query) {
+    return this.request('/addon/search', { gameId, pageSize: 3, searchFilter: query, sectionId: 4471, sort: 0 })
+  }
+
+  request (endpoint, queryParams = {}) {
+    const qParams = new URLSearchParams(queryParams)
+    return fetch(API_URL + endpoint + `?${qParams.toString()}`).then(res => res.json())
```
Comment (jaxxibae) [inline src/apis/CurseForge.js:19]: Use Axios, as per #1096.

File: src/commands/games/minecraft/modpack.js [hunk 5]
```diff
@@ -0,0 +1,36 @@
...
+    moment.locale(language)
+    channel.send(
+      new SwitchbladeEmbed(author)
+        .setThumbnail(attachments[0].url)
+        .setAuthor(name, this.embedLogoURL, websiteUrl)
+        .setColor(Constants.CURSEFORGE_COLOR)
+        .setDescription(summary)
+        .addField(t('commands:minecraft.subcommands.modpack.downloadCount'), downloadCount, true)
+        .addField(t('commands:minecraft.subcommands.modpack.releasedAt'), moment(dateReleased).format('LL'), true)
+        .addField('\u200B', `[${t('commands:minecraft.subcommands.modpack.directDownload')}](${latestFiles[0].downloadUrl})`)
```
Comment (jaxxibae) [inline src/commands/games/minecraft/modpack.js:33]: Replace `'\u200B'` with `Constants.EMPTY_SPACE`

**Review by jaxxibae — DISMISSED** [review]

File: src/apis/CurseForge.js (hunk 4 above)
Comment (Doges) [reply src/apis/CurseForge.js:19]: not required, as per https://github.com/SwitchbladeBot/switchblade/issues/1096#issuecomment-655563952

File: src/commands/games/minecraft/modpack.js (hunk 5 above)
Comment (Doges) [reply src/commands/games/minecraft/modpack.js:33]: kthen

**Review by almeidx — APPROVED** [review]
LGTM

**Review by metehus — APPROVED** [review]
SGFM

**Comment by Doges** [comment]
poggers

**Pull request merged**